app.js

```diff
@@ -40,35 +36,32 @@ app.use('/account', account);
 
 
 // catch 404 and forward to error handler
-app.use(function(req, res, next) {
-	'use strict';
-	var err = new Error('Not Found');
-	err.status = 404;
-	next(err);
+app.use((req, res, next)=> {
+  const err = new Error('Not Found');
+  err.status = 404;
+  next(err);
 });
 
 // error handlers
 // will print stacktrace
 if (app.get('env') === 'development') {
-	app.use(function(err, req, res, next) {
-		'use strict';
-		res.status(err.status || 500);
-		res.render('error', {
-			message: err.message,
-			error: err
-		});
-	});
+  app.use((err, req, res)=> {
```

Comment:
__[jshint]__
*  'arrow function syntax (=>)' is only available in ES6 (use esnext option).